scilpy/tracking/tracker.py

```diff
@@ -16,10 +16,14 @@
 from scilpy.tracking.seed import SeedGenerator
 
 # For the multi-processing:
-data_file_info = None
+# Iterable. Will contain all parameters necessary for a sub-process
+# initialization.
+multiprocess_init_args = []
```

Comment:
Couldn't it be a dictionary? Like, set to `None` here but pass a `dict` to the pool `init_args`?

Reply:
Just to increase the code readability. I just feel like having `np.load(init_args['data'], init_args['mmap_mode'])` would be clearer. But I won't defend my point with my life either, your call.